lib/taskana-core/src/main/java/pro/taskana/configuration/DbSchemaCreator.java

```diff
@@ -76,6 +78,39 @@ private boolean isSchemaPreexisting(ScriptRunner runner) {
         return true;
     }
 
+    public boolean isValidSchemaVersion(String expectedVersion) {
+        SqlRunner runner = null;
+        try {
+            Connection connection = dataSource.getConnection();
+            runner = new SqlRunner(connection);
+            LOGGER.debug(connection.getMetaData().toString());
+
+            String query = "select VERSION from TASKANA.TASKANA_SCHEMA_VERSION where ID = '1' AND VERSION = '"
```

Comment:
I believe this is wrong. I would expect a list of versions later on (each fix and upgrade leaving its signature) - therefore we should query for the largest number and compare that one.